src/translations/en.json

```diff
@@ -55,17 +55,35 @@
     }
   },
   "home":{
+    "howItWorks": {
+      "button": "How it works",
+      "title": "Looking To Play?",
+      "content": "Our weekly play sessions are open to everyone:",
+      "wednesday": "Wednesday nights at 8:30pm EST",
+      "thursday": "Thursdays at 12pm EST",
+      "findMatch": "Can't find a match? Join us on",
+      "discord": "discord",
```

Comment:
I think we can remove this since it's a name that we don't translate differently across languages
```suggestion
```